### PR TITLE
Language aware diff headers

### PR DESCRIPTION
**Pull request by BoboTiG**

### CODE REVIEW

**Comment by Legorooj** [comment]
Awesome idea - however can you please stick `[skip ci]` on the first line of the commit message? Half our pipelines ignore it otherwise.

File: .gitattributes [hunk 1]
```diff
@@ -25,3 +25,16 @@ runw_d	-text
...
+# https://tekin.co.uk/2020/10/better-git-diff-output-for-ruby-python-elixir-and-more
+# https://gist.github.com/tekin/12500956bd56784728e490d8cef9cb81
+# https://github.com/git/git/blob/master/userdiff.c
+*.c     diff=cpp
+*.h     diff=cpp
+*.md    diff=markdown
+*.py    diff=python
+*.rc    diff=pascal
+*.spec  diff=python
+*.tox   diff=python
```
Comment (BoboTiG) [inline .gitattributes:39]:
```suggestion
*.toc   diff=python
```

**Comment by BoboTiG** [comment]
Done @Legorooj. Still, the GitHub workflow does to respect the condition 🤔

**Comment by Legorooj** [comment]
Hmm. I'll get someone on that, thanks @BoboTiG.

**Comment by BoboTiG** [comment]
I can check that today or tomorrow, it should not be a big deal.

**Pull request merged**